examples/trusted_information_retrieval/client/rust/src/main.rs

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Client for the Trusted Information Retrieval example.
+
+pub mod proto {
+    tonic::include_proto!("oak.examples.trusted_information_retrieval");
+}
+
+use anyhow::anyhow;
+use log::info;
+use oak_abi::label::Label;
+use prost::Message;
+use proto::{
+    trusted_information_retrieval_client::TrustedInformationRetrievalClient,
+    ListPointsOfInterestRequest, Location,
+};
+use structopt::StructOpt;
+use tonic::{
+    metadata::MetadataValue,
+    transport::{Certificate, Channel, ClientTlsConfig},
+    Request,
+};
+
+#[derive(StructOpt, Clone)]
+#[structopt(about = "Trusted Information Retrieval Client")]
+pub struct Opt {
+    #[structopt(
+        long,
+        help = "URI of the Oak application to connect to",
+        default_value = "https://localhost:8080"
+    )]
+    uri: String,
+    #[structopt(
+        long,
+        help = "PEM encoded X.509 TLS root certificate file used by gRPC client",
+        default_value = ""
+    )]
+    grpc_client_root_tls_certificate: String,
+    #[structopt(
+        long,
+        help = "Requested location (latitude and longitude separated by space)",
```

Comment:
Why space separated rather than comma-separated?   

[Using a space raises some pesky questions about quoting and arguments etc.  Do you mean one command line argument with a space in it (`client --location "1 2"`), or does this consume two arguments (`client --location 1 2`) ?]

Reply:
Super-nit: lat/long wrt what?  WGS84?

Reply:
Or, you know, just use two separate flags :)

Reply:
Done
Added two separate flags =)